go/api/generic_commands.go

```diff
@@ -30,4 +30,49 @@ type GenericBaseCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/del/
 	Del(keys []string) (Result[int64], error)
+
+	// Renames key to new key.
+	//  If new Key already exists it is overwritten.
+	//
+	// Note:
+	//  When in cluster mode, both key and newKey must map to the same hash slot.
+	//
+	// Parameters:
+	//  key to rename.
+	//  newKey The new name of the key.
+	//
+	// Return value:
+	// If the key was successfully renamed, return "OK". If key does not exist, an error is thrown.
+	//
+	// Example:
+	//  result, err := client.Persist([]string{"key","newkey"})
+	//	if err != nil {
+	//	    // handle error
+	//	}
+	//	fmt.Println(result.Value()) // Output: string
```

Comment:
Please change `Output: string` to `Output: OK` for the other function as well.

Reply:
corrected pn my latest commit: 06c3e440684edeba748a71f0cc6f241a26b3740d